docker-compose.yml

```diff
@@ -36,7 +36,7 @@ services:
     ports:
       - 443:443
     volumes:
-      - ~/www:/var/www/html:rw,delegated
+      - ../:/var/www/html:rw,delegated
```

Comment:
It's possible to fix this in a variable ?

Reply:
Indeed, it should be possible!